CHANGELOG.md

```diff
@@ -3,6 +3,22 @@ aws-parallelcluster-cookbook CHANGELOG
 
 This file is used to list changes made in each version of the AWS ParallelCluster cookbook.
 
+2.6.2
+-----
+
+**ENHANCEMENTS**
+
+**CHANGES**
+- Upgrade EFA installer to version 1.8.4
+  - Kernel module: efa-1.5.1 (no change)
+  - RDMA core: rdma-core-25.0 (distributed only) (no change)
+  - Libfabric: libfabric-aws-1.9.0amzn1.1 (no change)
+  - Open MPI: openmpi40-aws-4.0.3 (vs. openmpi40-aws-4.0.2)
+  - EFA profile: efa-profile-1.0 (new)
+  - EFA config: efa-config-1.0 (new)
```

Comment:
do you know what are efa-profile and efa-config for?
I'd add an explanation for these two new files in the commit message, to remember us in the future why EFA added these new files and how are we using them.

Reply:
No. I just copied it from the email that subspace sent out to be honest.